### DISCUSSION TITLE
Controller is missing `defaultValue` prop when using `useFieldArray`

### DISCUSSION THREAD

**Discussion by Filta-leo** [react-hook-form/react-hook-form#7183] [General]
I am completely 100% copied  this code
https://codesandbox.io/s/react-hook-form-usefieldarray-forked-hyj85?file=/src/index.js

but on my  side, defaultValues are not rendering. resulting to Controller is missing `defaultValue` prop when using `useFieldArray` error.

What Im I missing?

**Comment by bluebill1049** [comment]
<img width="1049" alt="Screen Shot 2021-12-01 at 9 05 41 am" src="https://user-images.githubusercontent.com/10513364/144135633-c557c158-df25-4ea4-8c80-1a151b951677.png">

what's your issue here?
